src/uct/cuda/cuda_ipc/cuda_ipc_cache.c

```diff
@@ -305,7 +305,7 @@ UCS_PROFILE_FUNC(ucs_status_t, uct_cuda_ipc_map_memhandle, (key, mapped_addr),
                 }
             }
         } else {
-            ucs_debug("%s: failed to open ipc mem handle. addr:%p len:%lu",
+            ucs_error("%s: failed to open ipc mem handle. addr:%p len:%lu",
```

Comment:
looks like log level was "swapped" ith line 114..

Reply:
Should be fixed now. cc @pentschev